pom.xml

```diff
@@ -61,6 +68,12 @@
       <artifactId>spring-cloud-gcp-starter-pubsub</artifactId>
     </dependency>
 
+    <dependency>
```

Comment:
Btw, Is this necessary?

Reply:
I'm not sure. I'll check without it once